From d8dab08fe8db137c6bbd8bbdc3d9b01d98c48910 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 19 Dec 2019 13:37:49 +0100
Subject: [PATCH] account: Fix property name

Spaces are not valid characters in property names, and never were.
Until recently GLib silently fixed up the name by replacing the
space with '-', but now tightened up the validation.
---
 src/mcd-account.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git src/mcd-account.c src/mcd-account.c
index 8009110e..de5c61cf 100644
--- a/src/mcd-account.c
+++ b/src/mcd-account.c
@@ -3442,7 +3442,7 @@ mcd_account_class_init (McdAccountClass * klass)
 
     g_object_class_install_property
         (object_class, PROP_CONNECTIVITY_MONITOR,
-         g_param_spec_object ("connectivity monitor",
+         g_param_spec_object ("connectivity-monitor",
                               "Connectivity monitor",
                               "Connectivity monitor",
                               MCD_TYPE_CONNECTIVITY_MONITOR,
-- 
2.24.1

